tests/playwright/sanity/modules/promotions/admin-menu-items/custom-code-promotions.test.ts

```diff
@@ -0,0 +1,10 @@
+import { test } from '@playwright/test';
+import WpAdminPage from '../../../../pages/wp-admin-page';
+
+test( 'Custom code promotion screenshot', async ( { page }, testInfo ) => {
+	const wpAdminPage = new WpAdminPage( page, testInfo ),
+		promotionContainer = '.e-feature-promotion';
+	await wpAdminPage.login();
+	await wpAdminPage.promotionPageScreenshotTest( page, promotionContainer, 'elementor_custom_code', 'custom-code-menu-item-desktop' );
```

Comment:
Add the test as an additional step in the `promotions.test.ts` file.
don't open a new test file.

```suggestion
	await wpAdminPage.promotionPageScreenshotTest( promotionContainer, 'elementor_custom_code', 'custom-code-menu-item-desktop' );
```